admin/bin/qserv-configure.py

```diff
@@ -1,311 +1,381 @@
+        qserv template and symlink it
+        """
+        template_file = os.path.join(
+            self._template_root, "etc", filename
+        )
+        cfg_file = os.path.join(
+            self.args.qserv_run_dir, "etc", filename
+        )
+        configure.apply_tpl_once(
+            template_file,
+            cfg_file
+        )
+        _LOG.debug(
+            "Client configuration file created: {0}".format(cfg_file)
+        )
 
-        try:
-            logging.info(
-                "Reading meta-configuration file {0}".format(args.meta_config_file))
-            config = commons.read_config(args.meta_config_file)
+        if os.path.exists(symlink):
+            try:
+                is_symlink_correct = os.path.samefile(symlink, cfg_file)
+            except os.error:
+                # link is broken
+                is_symlink_correct = False
+
+            if not is_symlink_correct:
+                if self.args.force or configure.user_yes_no_query(
+                        "Do you want to update symbolic link {0} to {1}?".format(os.path.realpath(symlink),
+                                                                                 cfg_file)):
+                    os.remove(symlink)
+                    os.symlink(cfg_file, symlink)
+                else:
+                    _LOG.fatal("Symbolic link to client " +
+                               "configuration unmodified. Exiting.")
+                    sys.exit(1)
+        else:
+            try:
+                os.remove(symlink)
+                _LOG.debug("Removing broken symbolic link : {0}".format(symlink))
+            except os.error:
+                pass
+            os.symlink(cfg_file, symlink)
+
+        _LOG.info(
+            "{0} points to: {1}".format(symlink, cfg_file)
+        )
 
-            # used in templates targets comments
-            config['qserv']['meta_config_file'] = args.meta_config_file
+    def _template_to_client_config(self, product):
+        """
+        Create configuration files for a product from qserv_run_dir
+        templates files, and symlink it to client configuration
+        (for example ~/.lsst/ directory for Qserv product)
+        """
+        homedir = os.path.expanduser("~")
+        if product == configure.QSERV:
+            # might need to create directory first
+            try:
+                os.makedirs(os.path.join(homedir, ".lsst"))
+                _LOG.debug(
+                    "Creating client configuration directory : ~/.lsst")
+            except os.error:
+                pass
+            self._template_to_symlink("qserv-client.conf", os.path.join(homedir, ".lsst", "qserv.conf"))
+            self._template_to_symlink("logging.ini", os.path.join(homedir, ".lsst", "logging.ini"))
+        elif product == configure.MYSQL:
+            self._template_to_symlink("my-client.cnf",
+                                      os.path.join(homedir, ".my.cnf"))
```

Comment:
Should we get rid of `my.cnf` in home directory, I think we opened a ticket for it few months ago?